### PR TITLE
#8 Database Queries and Class implementation

### PR DESCRIPTION
**Pull request by anjali-rathod**
The functionality of Issue book code is left .

### CODE REVIEW

File: homepagecontent.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" type="text/css" href="aboutus.css">
+</head>
+<?php include 'homepage.php'; ?>
+
+<body>
+
+    <?php 
+    echo "<br><br><br><br><br>";
+    ini_set("display_errors", 1);
+    error_reporting(E_ALL);
```
Comment (guptahemant) [inline homepagecontent.php]: Remove this code, move it to credentials file.

File: homepagecontent.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?php session_start();
+    include "books.php"; ?>
```
Comment (guptahemant) [inline homepagecontent.php]: Where is the books.php file?

File: user.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?php 
+class user
```
Comment (guptahemant) [inline user.php]: use namespaces, Refer #5

File: register.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
+<?php 
+session_start();
+include "user.php";
```
Comment (guptahemant) [inline register.php]: utilize `use` statements to include the class, Refer PR for #5

**Review by guptahemant — CHANGES_REQUESTED** [review]

File: homepagecontent.php (hunk 2 above)
Comment (anjali-rathod) [reply homepagecontent.php]: Uploaded it in a new PR

File: homepagecontent.php (hunk 1 above)
Comment (anjali-rathod) [reply homepagecontent.php]: ok

File: user.php (hunk 3 above)
Comment (anjali-rathod) [reply user.php]: added

File: register.php (hunk 4 above)
Comment (anjali-rathod) [reply register.php]: added

**Review by guptahemant — CHANGES_REQUESTED** [review]
Please squash and rebase the commits and use the session management code added with #5

**Comment by guptahemant** [comment]
Please also include a databse class which will be utilized by other classes for connecting with database

**Comment by guptahemant** [comment]
@anjali-rathod, please include the changes suggested earlier in follow up PRs

**Pull request merged**